### PR TITLE
DBZ-6372 - Kinesis Sink - AWS Credentials Provider

### PR DESCRIPTION
**Pull request by brenoavm**

### CODE REVIEW

File: debezium-server-kinesis/src/main/java/io/debezium/server/kinesis/KinesisChangeConsumer.java [hunk 1]
```diff
@@ -78,8 +78,7 @@ void connect() {
         region = config.getValue(PROP_REGION_NAME, String.class);
         endpointOverride = config.getOptionalValue(PROP_ENDPOINT_NAME, String.class);
         final KinesisClientBuilder builder = KinesisClient.builder()
-                .region(Region.of(region))
-                .credentialsProvider(ProfileCredentialsProvider.create(credentialsProfile));
+                .region(Region.of(region));
```
Comment (Naros) [inline debezium-server-kinesis/src/main/java/io/debezium/server/kinesis/KinesisChangeConsumer.java:81]: With the removal of `ProfileCredentialsProvider` usage here, I suspect the import above is not used, could the unused imports also be removed as well?

Comment (brenoavm) [reply debezium-server-kinesis/src/main/java/io/debezium/server/kinesis/KinesisChangeConsumer.java:81]: Yes, we probably can remove the unused imports. I will commit it again.

**Comment by jpechane** [comment]
@brenoavm Thanks for the PR
IMHO to make sure that this feature is backward compaible we should apply a differnt approach.
Credentials profile property should be checked if it is present - instead of assuming it will be `default` if not. If present, then `ProfileCredentialsProvider` will be used, otherwise it will not. In that case it is also important to update the documentation to reflet the change.

**Review by Naros — APPROVED** [review]
Overall this LGTM, but could you please remove the merge commit @brenoavm ?
There should also be a sibling PR opened in the debezium/debezium repository updating the documentation accordingly.

**Comment by jpechane** [comment]
@brenoavm Squased and applied, thanks!

**Pull request merged**